app/presenters/return-logs/setup/meter-provided.presenter.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+/**
+ * Format data for the `/return-log/setup/{sessionId}/meter-provided` page
+ * @module MeterProvidedPresenter
+ */
+
+/**
+ * Format data for the `/return-log/setup/{sessionId}/meter-provided` page
+ *
+ * @param {module:SessionModel} session - The return log setup session instance
+ *
+ * @returns {object} page data needed by the view template
+ */
+function go(session) {
+  const {
+    id: sessionId,
+    data: { returnReference },
```

Comment:
You should be able to do this thanks to the way the model has been built:
```suggestion
    returnReference,
```

Reply:
This is going to be done in a tidy up PR:
https://github.com/DEFRA/water-abstraction-system/pull/1626